src/patterns.json

```
@@ -124,7 +124,7 @@
   "pageburst",
   "parser",
   "perl",
-  "phantom",
+  "phantom\\.?js",
```

```suggestion
  "phantomjs",
```

The only mention of "phantom.js" with the dot I found also includes the substring "bot"

https://github.com/ariya/phantomjs/blob/0a0b0facb16acfbabb7804822ecaf4f4b9dce3d2/examples/render_multi_url.js#L37

I'm going to recommend the string "phantomjs" instead